src/components/PlayerForm/PlayerForm.tsx

```diff
@@ -4,23 +4,43 @@ import "./playerForm.css";
 import AvatarPicker from "../AvatarPicker";
 import Button from "../Button";
 import FormError from "../FormError";
-import GenderSelect from "../GenderSelect";
+import SexSelect from "../SexSelect";
 import Input from "../Input";
 import Label from "../Label";
+import { Player } from "types/player";
 
-const PlayerForm = (props) => {
-  const [playerData, setPlayerData] = useState(props.defaultFormData || {});
-  const [errors, setErrors] = useState({});
+type FormErrors = {
+  name?: string;
+  sex?: string;
+  avatar?: string;
+};
+
+type Props = {
+  defaultFormData?: Partial<Player>;
```

Comment:
Instead of `Partial<Player>` you can look into use `Omit<Player, 'id', 'default_sex', 'blah'>` which may be more accurate, here!